org.gnome.SwellFoop.json

```diff
@@ -0,0 +1,45 @@
+{
+    "app-id" : "org.gnome.SwellFoop",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "swell-foop",
+    "rename-icon" : "swell-foop",
+    "rename-desktop-file" : "swell-foop.desktop",
```

Comment:
You might need to add a rename-appdata too. Otherwise the build will fail here on Flathub due to a missing appdata file.